pyproject.toml

```diff
@@ -1,29 +1,30 @@
-[tool.poetry]
-name = "myproj"
-version = "0.1.0"
-description = ""
-authors = ["Janne Enberg <janne@lieturd.com>"]
-
-[tool.poetry.dependencies]
-python = "^3.6"
-invoke = "^1.3"
-coloredlogs = "^10.0"
-pyyaml = "^5.1"
-
-[tool.poetry.dev-dependencies]
-pytest = "^5.2"
-codecov = "^2.0"
-
-[tool.isort]
-line_length = 88
-use_parentheses = true
-include_trailing_comma = true
-multi_line_output = 3
-
-# Not actually supported yet, future-proofing
-[tool.bandit]
-skips = "B101"
-
-[build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+[tool.poetry]
```

Comment:
I guess the line-endings were changed? Are they LF now?

Reply:
Yes, changed from CRLF to LF.